### DISCUSSION TITLE
Buffer function call in TFT_eSPI.cpp

### DISCUSSION THREAD

**Discussion by ShopCartz** [Bodmer/TFT_eSPI#2472] [General]
Firstly, thank you for the library.

I am using the latest version of the library 2.5.23 with a GC9A01_DRIVER and ESP32

I am having an issue with getbuffer. TFT_eSPI.cpp

  Note: ChatGPT has been writing the code and advising (I am not a programmer) , it has advised me to use the either of the following but 'class TFT_eSPI' has no member named 'getBuffer' or getFrameBuffer. 

 // Update display
  tft.pushImage(0, 0, tft.width(), tft.height(), tft.getBuffer()); // update display with TFT_eSPI buffer

// Update display
  tft.pushImage(0, 0, tft.width(), tft.height(), tft.getFrameBuffer()); // update display with TFT_eSPI frame buffer
  
  
Were the functions present in earlier version of the library? 

What should I be using instead?

Thanks for any advice.

**Comment by Bodmer** [comment]
Those functions are not part of TFT_eSPI and never have been.

What are you trying to do?
